### ISSUE TITLE
I think we need `babel-core` as a dependency here

### ISSUE THREAD

**Issue by lyzadanger** [cloudfour/core-gulp-tasks#24]
For `gulp` to work with a `babel`-ified `gulpfile`, `babel-core` is a dependency. I know that the repos that *use* this collection of stuff as an `npm` module typically have `babel-core` already, but for this repo to be standalone, I think it needs to be a dependency. 

See https://markgoodyear.com/2015/06/using-es6-with-gulp/

**Comment by erikjung** [comment]
@lyzadanger Yeah, I think we would need to add both `babel-core` and `babel-preset-es2015` to our `devDependencies` and then rename `test/gulpfile.js` to `test/gulpfile.babel.js`.

CC @mrgerardorodriguez

**Comment by erikjung** [comment]
@lyzadanger @mrgerardorodriguez We might also want to update the README to mention that Gulp 3.9 or higher is a requirement.
